### PR TITLE
New version: kalilinux.kalilinux version 1.13.1.0

### LINKED ISSUE #102537: [Update Request]: WSL Package updates
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

Canonical.Ubuntu.2204

### Package Version

2204.221101

### Please describe the changes you would like to see

Package | Version | Url
-|-|-
Canonical.Ubuntu.2204 | 2204.221101 | https://wslstorestorage.blob.core.windows.net/wslblob/Ubuntu2204-221101.AppxBundle
Debian.Debian | 1.12.2.0 | https://wslstorestorage.blob.core.windows.net/wslblob/TheDebianProject.DebianGNULinux_1.12.2.0_neutral___76v4gfsz19hv4.AppxBundle
kalilinux.kalilinux| 1.13.1.0 | https://wslstorestorage.blob.core.windows.net/wslblob/KaliLinux_1.13.1.0.AppxBundle

Source: https://learn.microsoft.com/en-us/windows/wsl/install-manual#downloading-distributions

### PR DESCRIPTION
**Pull request by Dragon1573**
## Description

- Resolve #102537

## Checks

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/131543)

### CODE REVIEW

**Comment by Dragon1573** [comment]
There's a https://github.com/microsoft/winget-pkgs/labels/Validation-Domain error without any reason from bot account.

Could contributors or moderators help investigate this problem?

**Comment by stephengillie** [comment]
Hi @Dragon1573,

Sorry for the delay. I couldn't navigate to the InstallerUrl from the PackageUrl. Can you help me to find the link?

**Comment by Dragon1573** [comment]
I can't find it either ...

The `installerUrl` seems to come from [Microsoft Learn](https://learn.microsoft.com/en-us/windows/wsl/install-manual#downloading-distributions) mentioned in https://github.com/microsoft/winget-pkgs/issues/102537#issue-1668999292.

Shall I update the `packageUrl`?

File: manifests/k/kalilinux/kalilinux/1.13.1.0/kalilinux.kalilinux.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: kalilinux.kalilinux
+PackageVersion: 1.13.1.0
+PackageLocale: en-US
+Publisher: Kali Linux
+Author: Kali Linux
+PackageName: Kali Linux
+PackageUrl: https://www.kali.org/tutorials/kali-on-the-windows-subsystem-for-linux/
```
Comment (stephengillie) [inline manifests/k/kalilinux/kalilinux/1.13.1.0/kalilinux.kalilinux.locale.en-US.yaml:10]:
```suggestion
PackageUrl: https://learn.microsoft.com/en-us/windows/wsl/install-manual#downloading-distributions
```
Yes, the InstallerUrl should be available on the PackageUrl or a few clicks away.

**Comment by Dragon1573** [comment]
> [!NOTE]
>
> This pull request requires a validation on Azure Pipeline, but @wingetbot didn't trigger it automatically. Moderators please help trigger the pipeline manually. Thank you! ❤️

**Comment by stephengillie** [comment]
> Note
> 
> This pull request requires a validation on Azure Pipeline, but @wingetbot didn't trigger it automatically. Moderators please help trigger the pipeline manually. Thank you! ❤️

Please feel free to @ me in the future.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**